src/diffusers/schedulers/scheduling_sde_ve.py

```diff
@@ -98,6 +100,9 @@ def get_adjacent_sigma(self, timesteps, t):
         raise ValueError(f"`self.tensor_format`: {self.tensor_format} is not valid.")
 
     def set_seed(self, seed):
+        warnings.warn(
+            "The method `set_seed` is outdated. Please consider passing a generator instead.", DeprecationWarning
```

Comment:
Maybe suggest it will be removed in the future (we can use whatever version you think would be reasonable)

```suggestion
            "The method `set_seed` is deprecated and will be removed in version `0.4.0`. Please consider passing a generator instead.", DeprecationWarning
```